Tools/machines/perlmutter-nersc/Containerfile

```
@@ -210,8 +180,11 @@ RUN python3 -m venv /opt/venv && \
         matplotlib       \
         numpy            \
         openpmd-api      \
+        git+https://github.com/RemiLehe/openPMD-viewer.git@get_laser_spectral_intensity  \
         pandas           \
+        pymongo          \
         scipy            \
+        git+https://github.com/RemiLehe/transparent_imshow.git \
```

Simpler for base image:
```suggestion
        openpmd-viewer   \
        pandas           \
        scipy            \
```